### ISSUE TITLE
How many gpus were used ?

### ISSUE THREAD

**Issue by ksh11023** [TRAILab/CaDDN#60]
hi, 
Thank you for sharing your work. 

when training on Kitti benchmark, do you mean by  batch size of 4 per GPU?
which equals to batch size of 8 when using two GPUs? Am I getting this right..?

**Comment by codyreading** [comment]
Yup that's correct. I used the V100 GPU (32G), so I was able to fit those batch sizes in memory. You can always decrease the batch size if you need to fit with less GPU memory.

**Comment by ksh11023** [comment]
Thank you for your reply!!

So, do I have to use batch size of 8 for Kitti benchmark, to get the results on the paper? 
I am having trouble getting the same(similar) results using batch size of 4 on Kitti benchmark.

**Comment by codyreading** [comment]
The results on the paper are all reported with a batch size of 4, so this should be fine for any of your experiments. What results are you getting for your models?

**Comment by ksh11023** [comment]
When training on Kitti train set(3712) and validating on Val set(3769) 

In the paper it says:
Car (IOU=0.7)
easy: 23.57 mod: 16.31 hard: 13.84 

And my results are :
Car(IOU=0.7)
easy: 21.89 mod: 14.58 hard:12.95 

I used batch size 4, trained from the scratch with other settings same as the paper.

**Comment by codyreading** [comment]
That level of variance is expected between runs for CaDDN. Please see [here ](https://github.com/open-mmlab/OpenPCDet/issues/581)f or a more in depth discussion about the reproducibility. If you would like to achieve better results, I recommend performing multiple runs and select the one with the best results.

**Comment by ksh11023** [comment]
Cody thank you for the swift reply!

I'll follow your advice. Thanks again for your great work:)
